syntax = "proto3";

package envoy.extensions.filters.http.upstream_codec.v3;

import "udpa/annotations/status.proto";

option java_package = "io.envoyproxy.envoy.extensions.filters.http.upstream_codec.v3";
option java_outer_classname = "UpstreamCodecProto";
option java_multiple_files = true;
option go_package = "github.com/envoyproxy/go-control-plane/envoy/extensions/filters/http/upstream_codec/v3;upstream_codecv3";
option (udpa.annotations.file_status).package_version_status = ACTIVE;

// [#protodoc-title: UpstreamCodec]
// Codec :ref:`configuration overview <config_http_filters_upstream_codec>`.
// [#extension: envoy.filters.http.upstream_codec]

message UpstreamCodec {
}
